api/src/test/java/com/code4ro/nextdoor/emergency/contact/service/EmergencyContactServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package com.code4ro.nextdoor.emergency.contact.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import com.code4ro.nextdoor.emergency.contact.dto.EmergencyContactDto;
+import com.code4ro.nextdoor.emergency.contact.entity.EmergencyContact;
+import com.code4ro.nextdoor.emergency.contact.repository.EmergencyContactRepository;
+import com.code4ro.nextdoor.emergency.contact.service.impl.EmergencyContactServiceImpl;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class EmergencyContactServiceImplTest {
+
+    private static final String ID = "14022837-be98-4457-af25-075d7a136a33";
+    private static final UUID UID = UUID.fromString(ID);
+
+    @InjectMocks
+    private EmergencyContactServiceImpl underTest;
+
+    @Mock
+    private EmergencyContactRepository emergencyContactRepository;
+
+    private static EmergencyContact emergencyContact =
+        EmergencyContact.builder()
+        .address("address")
+        .email("email@something.com")
+        .name("name")
+        .surname("surname")
+        .telephoneNumber("07xx43x56x")
+        .build();
+
+    @Test
+    public void testFindAll() {
+        when(emergencyContactRepository.findAll()).
+            thenReturn(Arrays.asList(emergencyContact));
+
+        List<EmergencyContactDto> result = underTest.findAll();
+
+        assertEquals(1, result.size());
```

Comment:
Pentru partea de assert eu as merge cu https://joel-costigliola.github.io/assertj/ mi se pare mai usor de folosit si are multe chestii faine care fac partea de verificare mai usoara 

Daca folosit assertj toate asserturile ar fi 
```java
asserThat(result).hasSize(1).containsOnly(Collections.singletonList(emergencyContact));
```

Reply:
In PR-ul pentru #21, tot pe AssertJ am mers, mi se pare cel mai elegant dintre junit/hamcrest/assertj